packages/scripts/package.json

```diff
@@ -34,8 +34,10 @@
 		"@wordpress/babel-preset-default": "file:../babel-preset-default",
 		"@wordpress/jest-preset-default": "file:../jest-preset-default",
 		"@wordpress/npm-package-json-lint-config": "file:../npm-package-json-lint-config",
+		"babel-eslint": "8.0.3",
```

Comment:
I wanted to bump `babel-eslint` and `eslint` but it throws an error with the latest version. We should tackle it seperately.